### DISCUSSION TITLE
Does RHF order errorMessages as default.

### DISCUSSION THREAD

**Discussion by trondal** [react-hook-form/react-hook-form#12025] [Q&A] [answered]
Hi, and thanks for a fantastic project. 

Does RHF order the errormessages in any way? I tried to read to the source code, but could not find any ordering. However it sees that RHF order the errormessages according to property name. Is this correct? And if so, is there any way to "turn off" ordering? Or should I just write an iterator of some sort where I send inn the correct order?

**Comment by tatsmaki** [comment, marked as answer]
Object iteration order may vary across browsers and ECMAScript versions.
If you use `Object.keys` or `for in` to iterate over an object's fields, you should implement your own sorting to maintain desired order.

**Reply by trondal** [reply]
Thank you for a very good answer, and the short reply time. Kudos to you👍
